training/tests/integration/conftest.py

```diff
@@ -176,3 +177,18 @@ def gnn_config(
     cfg = OmegaConf.merge(template, testing_modifications_with_temp_dir, use_case_modifications)
     OmegaConf.resolve(cfg)
     return cfg, dataset_urls[0]
+
+
+@pytest.fixture
+def gnn_config_with_checkpoint(gnn_config: OmegaConf, get_test_data: callable) -> OmegaConf:
+    cfg, dataset_url = gnn_config
+    existing_ckpt = get_test_data(
```

Comment:
Hi Vera, could you check if the implementation here is in line with your latest improvement in https://github.com/ecmwf/anemoi-core/commit/8a2b0c8ac05412944859b65485a5af7916ac5c9a ?

Reply:
Thank you Vera, I've overseen that `get_test_data` is passed as a fixture.

Great PR!